mackup/applications/i3wm.cfg

```diff
@@ -0,0 +1,7 @@
+[application]
+name=i3wm
+
+[configuration_files]
+.i3/config
+.i3status.conf
+.xsession
```

Comment:
Yeah, you're right, its not i3 specific, however its good to keep with i3wm configs.